python/lgbserver/poetry.lock

```diff
@@ -1236,7 +1254,6 @@ python-dateutil = "^2.8.0"
 pyyaml = "^6.0.0"
 requests = {version = "^2.32.2", optional = true}
 six = "^1.16.0"
-starlette = ">=0.40.0,<0.41.0"
```

Comment:
is this expected?

Reply:
Yes. We aren't directly depend on starlette. I think somebody forgot to update lock files.